hibernate-core/src/test/java/org/hibernate/orm/test/type/UUIDTypeConverterTest.java

```diff
@@ -130,7 +130,7 @@ public static class Id {
 	@Table(name = "TEST_IMAGE")
 	public static class Image extends Id {
 
-		@Column(unique = true, length = 16, nullable = false)
```

Comment:
I think I would rather skip this test for Informix if it doesn't support `unique` constraints on `binary` columns.

Reply:
Ok